concrete/src/File/Filesystem.php

```diff
@@ -91,11 +91,12 @@ public function getRootFolder()
      *
      * @param FileFolder $folder The parent folder
      * @param string $name The name of the new folder
+     * * @param int|\Concrete\Core\Entity\File\StorageLocation\StorageLocation|null $storageLocationID Id or object of the storage location of the new folder
```

Comment:
I'd start with the type that is preferred / most likely. Is that `int`? I also think the first `*` needs to be removed.

Reply:
Yes `int` is the most likely one.